Core.ApplicationServices/System/ItSystemService.cs

```diff
@@ -349,7 +350,7 @@ public bool CanCreateSystemWithName(int organizationId, string name)
                     return system;
                 }
 
-                var addReferenceResult = _referenceService.AddReference(systemId, ReferenceRootType.System, "Reference", string.Empty, urlReference);
+                var addReferenceResult = _referenceService.AddReference(systemId, ReferenceRootType.System, new ExternalReferenceProperties("Reference", string.Empty, urlReference, false));
```

Comment:
parameter should be "true" because then you don't have to set the master reference afterwards in line 359